xxl-rpc-core/src/main/java/com/xxl/rpc/core/remoting/net/impl/netty_http/server/NettyHttpServerHandler.java

```diff
@@ -147,4 +148,4 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
     }
 
-}
\ No newline at end of file
+}
```

Comment:
I edited this file in GitHub UI, not sure why it added this trailing newline, let's see if I can remove it(?)

```suggestion
}
```